### DISCUSSION TITLE
Does the raman coefficient Cr for SSMF based on depolarized pump or signal?

### DISCUSSION THREAD

**Discussion by caffery-chen** [Telecominfraproject/oopt-gnpy#477] [Q&A] [answered]
Dear contributors,

After I compared the Cr value in the GNpy code and some literatures, sometimes, there is an approx. a factor of 2 difference.

May I ask if the Cr coefficient has considered the depolarization? 

Thanks.
Caffery

**Comment by AndreaDAmico** [comment, marked as answer]
Dear @caffery-chen,

As a reference we have considered the Raman coefficient measurements presented in _Pincemin, Erwan, et al. "Raman gain efficiencies of modern terrestrial transmission fibers in S-, C-and L-band." Nonlinear Guided Waves and Their Applications. Optica Publishing Group, 2002._
In Fig.2 top right plot, the measured C-band Raman coefficient is shown for various fiber types. The measurement has been performed using a counter-propagating (citing) "polarization independent DRA". Therefore, we consider these measurements as reference for the depolarized Raman coefficient, which is the value that has to be used in GNPy. In particular, you can compare the SSMF measurement in Fig.2 top right plot, which is the lowest line with maximum around 0.4 (W * km)^-1, with the attached plot of the Raman coefficient used in GNPy.
In any case, can you kindly share the references where you have found this 2 factor difference?
![raman_coefficient](https://github.com/Telecominfraproject/oopt-gnpy/assets/39195346/ec6b7b36-69e6-48c0-a2ca-107b433019d8)

**Comment by caffery-chen** [comment]
Dear Andrea,

Thanks for your detailed explanation.

Following figure in Agrawal's Nonlinear Fiber Optics Third Edition
![image](https://github.com/Telecominfraproject/oopt-gnpy/assets/12408183/990b182f-7b5d-42ce-beba-9e9c68dffce8)
The Ref 10 is  R. H. Stolen and E. P. Ippen, Appl. Phys. Lett. 22, 276 (1973).

Maybe fused silica is somehow different to SSMF?

**Reply by caffery-chen** [reply]
Another reference:
Raman Amplification: Wayne S. Pelouch, _An Enabling Technology for Long-Haul Coherent Transmission Systems_, JLT VOL. 34, NO. 1, JANUARY 1, 2016

![image](https://github.com/Telecominfraproject/oopt-gnpy/assets/12408183/a224e3c8-e2cd-4f8f-b999-1488438832c4)

**Reply by caffery-chen** [reply]
After I check the Wayne S. Pelouch's JLT paper more closely, the above Fig.2 does include a factor of 2, since the Eq. 6 in the paper has a two in the denominator such that for depolarized case. the gr/Aeff shall be half of what is shown in Fig.2.
